### PR TITLE
SDK-193: Support for oracle opc cloud cluster operations. And image overrides.

### PR DESCRIPTION
**Pull request by tarrygo**
2 Changes are being pushed as part of this PR:
1. Support for Oracle opc cluster v2 APIs.
2. Adding support for internal image override.

### CODE REVIEW

File: qds_sdk/cloud/oracle_opc_cloud.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        Args:
+            username: Username for customer's oracle opc account. This
+                is required for creating the cluster.
+
+            password: Password for customer's oracle opc account. This
+                is required for creating the cluster.
+
+            rest_api_endpoint: Rest API endpoint for customer's oracle opc cloud.
+
+            use_account_compute_creds: Set it to true to use the account’s compute
```
Comment (tanishgupta1) [inline qds_sdk/cloud/oracle_opc_cloud.py]: fix this bad character ascii issue .Use '.

File: qds_sdk/cloud/oracle_opc_cloud.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    '''
+    qds_sdk.cloud.OracleOpcCloud is the class which stores information about oracle opc cloud config settings.
+    The objects of this class can be use to set oracle opc cloud_config settings while create/update/clone a cluster.
+    '''
+
+    def __init__(self):
+        self.compute_config = {}
+        self.network_config = {}
+        self.storage_config = {}
+
```
Comment (tanishgupta1) [inline qds_sdk/cloud/oracle_opc_cloud.py]: there is no location_config for this ?

**Comment by tanishgupta1** [comment]
LGTM.

File: qds_sdk/cloud/oracle_opc_cloud.py (hunk 2 above)
Comment (tarrygo) [inline qds_sdk/cloud/oracle_opc_cloud.py]: Yes there is no location lonfig, We define region by rest_api_endpoints and that is part of creds.

File: qds_sdk/clusterv2.py [hunk 3]
```diff
@@ -509,6 +517,12 @@ def cluster_info_parser(argparser, action):
                                    default=None,
                                    help="overrides for airflow cluster", )
 
+        internal_group = argparser.add_argument_group("internal settings")
```
Comment (sourabh912) [inline qds_sdk/clusterv2.py]: Should the internal settings be exposed to user? I believe for internal settings, one requires super admin access which an outside user won't have.

Comment (tarrygo) [inline qds_sdk/clusterv2.py]: Cluster V2 API supports adding the image overrides if it is provided, and it doesnot require super admin access.

**Comment by sourabh912** [comment]
Merged into `unreleased` branch. Commit: https://github.com/qubole/qds-sdk-py/commit/3e74dcbea67ed4f3731146e71b05077c20fd2776

**Pull request closed without merging**